pkg/kv/bulk/sst_adder.go

```diff
@@ -345,7 +345,7 @@ func addStatsToSplitTables(left, right, original *sstSpan, sendStartTimestamp ti
 		return err
 	}
 	statsIter.SeekGE(storage.MVCCKey{Key: right.start})
-	right.stats, err = storage.ComputeStatsForIter(statsIter, sendStartTimestamp.Unix())
+	right.stats, err = storage.ComputeStatsForIter(statsIter, sendStartTimestamp.UnixNano())
```

Comment:
I wonder if we could pass this a time.Time or at least a typed i64; this kind possible mistake makes me sad.

Reply:
>I wonder if we could pass this a time.Time or at least a typed i64; this kind possible mistake makes me sad.

I actually tried to do that before sending this out for review. It's possible, but ended up being non-trivial to make all of the public functions accept time.Time because some of the KV code has utc nanos on protobufs and it felt weird having to re-synthesize a time from that.